AWSCore/Logging/AWSDDOSLogger.m

```diff
@@ -55,16 +55,16 @@ - (void)logMessage:(AWSDDLogMessage *)logMessage {
         
         switch (logMessage->_flag) {
             case AWSDDLogFlagError     :
-                os_log_error(OS_LOG_DEFAULT, msg);
+                os_log_error(OS_LOG_DEFAULT, "%s", msg);
```

Comment:
Oh, neat. Never knew about that! Based on the limited searching I've done, it looks like this would be a good change.